webpack.config.js

```diff
@@ -13,6 +13,8 @@ const cloneDeep = require('lodash.clonedeep');
 // const CompressionPlugin = require('compression-webpack-plugin');
 const glob = require('glob');
 const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
+const NormalModuleReplacementPlugin = require('webpack/lib/NormalModuleReplacementPlugin');
```

Comment:
These two packages were missing when running this template using Embark 4 alpha. Mostly likely a problem with Embark 4 but at the time of writing this comment, this template doesn't work with it, so we need to keep an eye on this.

Reply:
**I think too: it seems to be a problem related to Embark 4.**

I configured a breakpoint in `webpack.config.js` to check the value of `process.env.NODE_PATH`:

* with embark v3.2.3:
  ```js
  process.env.NODE_PATH === '/home/steven/.nvm/versions/node/v8.11.3/lib/node_modules/embark/node_modules'
  ```
* with embark v4.0.0-alpha.1:
  ```js
  process.env.NODE_PATH === undefined
  ```

**May embark-framework/embark#1039 will fix this issue?**

Reply:
Sorry for the super late update here ... I'm playing around with this now and can confirm that it work with latest master :) (4.0.0-alpha.1)

Reply:
Great! Can we resolve this conversation?